invokeai/backend/stable_diffusion/diffusion/shared_invokeai_diffusion.py

```diff
@@ -191,43 +190,28 @@ def do_diffusion_step(
                 )
             )
 
-        wants_cross_attention_control = len(cross_attention_control_types_to_do) > 0
-        wants_hybrid_conditioning = isinstance(conditioning, dict)
-
-        if wants_hybrid_conditioning:
-            unconditioned_next_x, conditioned_next_x = self._apply_hybrid_conditioning(
-                x, sigma, unconditioning, conditioning
-            )
-        elif wants_cross_attention_control:
-            (
-                unconditioned_next_x,
-                conditioned_next_x,
-            ) = self._apply_cross_attention_controlled_conditioning(
+        if "default" in conditioning_info:
```

Comment:
this needs to support .swap() and also sequential guidance too

Reply:
it's just moved inside `_apply_standard_conditioning`